public/components/weather.jsx

```diff
@@ -47,25 +210,96 @@ class Weather extends React.Component {
   }
 
   render() {
-  	console.log(this.props.params);
     return (
       <div>
         <SearchBar />
-        <p>You choose a city. We'll forecast.</p>
         <WeatherCard
           cityname={this.state.cityname}
```

Comment:
Maybe add "current" as a prefix to all these to make it easier to understand that it's data for the current weather :1st_place_medal: